docs/current/sdk/python/459708-index.md

```diff
@@ -0,0 +1,69 @@
+---
+slug: /sdk/python
+---
+
+# Dagger Python SDK
```

Comment:
This page is a similar template to the others. I'm not sure if we want to change it or if there is value in following the same template for each SDK. WDYT @shykes @mircubed @jpadams ?

Reply:
I think we should keep it similar since a Python person might not check out the other SDK overviews, so they could miss the general context.